source/game.broadcastmaterial.programme.bmx

```diff
@@ -343,7 +343,7 @@ Type TProgramme Extends TBroadcastMaterialDefaultImpl {_exposeToLua="selected"}
 		data.RefreshTrailerTopicality()
 
 		If licence And licence.effects
-			Local effectParams:TData = New TData.AddInt("programmeId", Self.licence.GetId())
+			Local effectParams:TData = New TData.AddInt("programmeId", Self.licence.GetId()).AddNumber("playerID", Self.licence.GetOwner())
```

Comment:
AddInt() sollte hier "besser" sein ... ist ja eine Ganzzahl

Reply:
AddNumber hatte ich von einer anderen Stelle für add playerID übernommen. Ändere ich aber.

Reply:
```
	Method AddNumber:TData(key:Object, data:Double)
		Local dd:TDoubleData = New TDoubleData
		dd.value = data
		Add( key, dd )
		Return Self
rem

		If Double(Long(data)) = data 
			Return AddLong(key, Long(data))
		EndIf

		Return AddDouble(key, data)
endrem
	End Method
```
Du siehst, dass es mal eine Zeit gab, in der "AddNumber" versucht hat automatisch den Typus zu erkennen...

https://github.com/TVTower/TVTower/commit/d43119835a972f1cde21e426a02bc4a8fdb14c31
-> * Revert TData.AddNumber() for now until properly replaced

AddNumber() haette an sich funktionieren koennen, ich weiss aber nicht, woran es hier scheiterte.

Was aber an sich funktionieren sollte -- ist einfach einen Overload fuer die Typen anzubieten, dann ersparen wir uns das "ueberlegen".